src/controllers/UserController.js

```diff
@@ -147,6 +147,77 @@ class UserController {
     }
   }
 
+  /**
+   *
+   * @description method that updates user's profile settings
+   * @static
+   * @param {object} req HTTP Request object
+   * @param {object} res HTTP Response object
+   * @returns {object} HTTP Response object
+   * @memberof ProfileController
+   */
+  static async updateProfile(req, res) {
+    const userId = req.params.id;
+    try {
+      const user = await User.findOne({ where: { id: userId } });
```

Comment:
Since we are looking for an id which is the primary key, use the `findByPk` method instead. check it out in the sequelize docs.